### DISCUSSION TITLE
GitHub student certifications have consistently failed.

### DISCUSSION THREAD

**Discussion by xdouworld** [community/community#52242] [GitHub Education] [answered]
I have applied for githup student certification many times. But they all failed, and at first they would tell me what was wrong. But now, tell me this. Common verification problems and suggested solutions are documented here:  https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-c ampus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved. This makes me very confused, don't know what to change? Unfortunately, I don't have an education email. I wonder if this is the problem? Do you have to have an education mailbox?

**Comment by samus-aran** [comment, marked as answer]
Hi @xdouworld 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍

**Reply by xdouworld** [reply]
Thank you very much for your reply
I think I have solved this problem. I tried to re-register an account and modify my profile. fork a few high star items so that the machine validates that I don't want a robot. After waiting three days, I tried again and passed. And I tried my old account again. Luckily, I got the email. Not so quick to reject me. Looks like a manual certification.
I hope I can pass tomorrow.
Finally, have a nice day

**Comment by ardaastam** [comment]
Hi everyone,

I am not in the campus right now but ı need to join GitHub education package? I tried many times and I only got failed message from GitHub about my location. What should I do?
